src/components/SatelliteManifestPanel/SatelliteManifestPanel.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { FunctionComponent, useState, useRef, createRef } from 'react';
+import React, { FunctionComponent, useState, useReducer, useRef } from 'react';
```

Comment:
I would remove this `useReducer`. It seems like the only role it has here is to force a component update and this should be done with setRowExpandedStatus.  Whenever state is updated using a useState hook, it automatically rerenders.